talk/morelanguage.tex

```diff
@@ -118,6 +118,29 @@
   \end{exampleblock}
 \end{frame}
 
+\begin{frame}[fragile]
+  \frametitleii{Static Assertions}
+  \begin{block}{static\_assert declaration}
+    \begin{itemize}
+    \item Performs compile time assertions; meaning a failed assertion stops compilation
+    \item The expression evaluated has to be a constexpr boolean expression
+    \item Purely evaluated at compile time, no effect at runtime
+    \item Often used in template programming to make assertion on types, can be
+      used to validate boolean compile time expressions
+    \end{itemize}
+  \end{block}
+  \pause
+  \begin{exampleblock}{Example}
+    \begin{cppcode*}{}
+      constexpr int f(int x) {
+        return x > 1 ? x * f(x - 1) : 1;
+      }
+      static_assert(f(5)==120,"Expected f(5) to be 120!");
+    \end{cppcode*}
+  \end{exampleblock}
+\end{frame}
```

Comment:
It does fit this time, but barely. Easy to fix this time, by having the second bullet on a single line : "The expression has to be a constexpr boolean expression". I would also probably put the second part of the first point as a subbullet, gaining both a bit of space and readibility